cdap-integration-test/src/main/java/co/cask/cdap/test/remote/RemoteApplicationManager.java

```diff
@@ -193,58 +105,8 @@ public void waitForFinish(long timeout, TimeUnit timeoutUnit) throws TimeoutExce
   @Override
   public WorkflowManager startWorkflow(final String workflowName, Map<String, String> arguments) {
     // currently we are using it for schedule, so not starting the workflow
-    return new WorkflowManager() {
-      @Override
-      public List<ScheduleSpecification> getSchedules() {
-        try {
-          return getScheduleClient().list(application.getId(), workflowName);
-        } catch (Exception e) {
-          throw Throwables.propagate(e);
-        }
-      }
-
-      @Override
-      public List<RunRecord> getHistory() {
-        try {
-          return getProgramClient().getProgramRuns(application.getId(), ProgramType.WORKFLOW,
-                                                   workflowName, "ALL", 0, Long.MAX_VALUE, Integer.MAX_VALUE);
-        } catch (Exception e) {
-          throw Throwables.propagate(e);
-        }
-      }
-
-      public ScheduleManager getSchedule(final String schedName) {
-        return new ScheduleManager() {
-          @Override
-          public void suspend() {
-            try {
-              getScheduleClient().suspend(application.getId(), schedName);
-            } catch (Exception e) {
-              throw Throwables.propagate(e);
-            }
-          }
-
-          @Override
-          public void resume() {
-            try {
-              getScheduleClient().resume(application.getId(), schedName);
-            } catch (Exception e) {
-              throw Throwables.propagate(e);
-            }
-          }
-
-          @Override
-          public String status(int expectedCode) {
-            try {
-              return getScheduleClient().getStatus(application.getId(), schedName);
-            } catch (Exception e) {
-              throw Throwables.propagate(e);
-            }
-          }
-        };
-      }
-
-    };
+    Id.Program workflowId = Id.Program.from(application, ProgramType.WORKFLOW, workflowName);
+    return new RemoteWorkflowManager(workflowId, clientConfig, this);
```

Comment:
In this PR, I'm going to leave this behavior as is, because there is code that depends on it working this way.
https://github.com/caskdata/cdap/pull/2705/files#diff-0147c2712afdf71eec4b4fc988a99db7R107

I think we can remove it if we introduce a `getManager` method, without starting the program, but that will be a whole new set of APIs, and I don't want to mix that in here.